migration.md

```diff
@@ -1,67 +1,67 @@
-# Migration steps from `cortex-client` to `cortex-python`
+# Migration steps from `sensa-client` to `sensa-python`
```

Comment:
We should probably remote these existing `Migration steps` and other `Deprecation notes` to avoid confusing ourselves with backwards compatibility requirements. 

My understanding is `sensa-python` doesn't have to be backwards compatible with `cortex-python`, so the earlier migration notes from `cortex-client` -> `cortex-python` are probably less relevant. We probably just need a short note saying that users of `cortex-python` should instead install `sensa-python` and rename instances of `cortex` to `sensa`..